libprecompiled/UpdateContractStatusPrecompiled.cpp

```diff
@@ -65,31 +76,226 @@ bool UpdateContractStatusPrecompiled::checkAddress(Address const& contractAddres
     }
     catch (...)
     {
+        PRECOMPILED_LOG(ERROR) << LOG_BADGE("UpdateContractStatusPrecompiled")
+                               << LOG_DESC("address invalid") << LOG_KV("address", contractAddress);
         isValidAddress = false;
     }
     return isValidAddress;
 }
 
+ContractStatus UpdateContractStatusPrecompiled::getContractStatus(
+    ExecutiveContext::Ptr context, std::string const& tableName)
+{
+    Table::Ptr table = openTable(context, tableName);
+    if (table)
+    {
+        int status = ContractStatus::Available;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/manual/fisco/FISCO-BCOS/pullRequest?prid=4839130)